containers/concent-api/install-dependencies.sh

```diff
@@ -2,22 +2,41 @@
 
 # Update package lists for Debian and install dependencies
 dependencies=(
-    python3-pip
-    python3-virtualenv
-    virtualenv
-
     # Required by psycopg2 python package (PostgreSQL support)
     libpq-dev
 
     # Packages needed to build python extensions, dependencies of golem-messages in particular
     gcc
-    python3-dev
     libssl-dev
     git
+    pkg-config
+    libsecp256k1-dev
+    libffi-dev
+    software-properties-common
 )
+
+testing_dependencies=(
+    # We need to use python3.6. There is no official python3.6 package on the debian stretch,
+    # we must include test packages.
+    python3.6
+    python3.6-dev
+    virtualenv
+)
+
+apt-get --assume-yes update
+apt-get install              \
+    --assume-yes             \
+    --no-install-recommends  \
+    ${dependencies[*]}
+add-apt-repository "deb http://ftp.de.debian.org/debian testing main"
 apt-get --assume-yes update
```

Comment:
This is not the best way to install stuff from testing. Any apt command after doing this will install packages from testing (even if you don't specify `--target-release testing` explicitly). This now works but it's easy for someone to come later and break it.

You should use pinning instead. Add testing repo with piority < 100. Then use `--target-release`
See https://askubuntu.com/questions/27362/how-to-only-install-updates-from-a-specific-repository/814936#814936